en-US/firefox/channel/index.lang

```diff
@@ -99,6 +100,12 @@ Build, test, scale and more with the only browser built just for developers.
 Firefox Developer Edition automatically sends feedback to Mozilla.
 
 
+## TAG: android_nightly_copy
+;Check out new Android features in their earliest stages. Enjoy at your own risk.
+Check out new Android features in their earliest stages. Enjoy at your own risk.
+
+# This string is obsolete. Do not remove.
+# This string is used as a fallback.
```

Comment:
Remove both these comments. In this case the string is already (and still) used for desktop, is used a fallback only for Android, no need to tell localizers.